code/drasil-docLang/lib/Drasil/Sections/SpecificSystemDescription.hs

```diff
@@ -166,6 +193,9 @@ inModelIntro r1 r2 r3 r4 = foldlSP [S "This", phrase section_,
   pluralNP $ symbol_ `inThePP` model, S "identified in", namedRef r3 (plural thModel) `S.and_`
   namedRef r4 (plural genDefn)]
 
+inModelIntroNoContent :: Section -> Section -> Section -> Section -> Contents
+inModelIntroNoContent _ _ _ _ = mkParagraph $ S "There are no instance models."
```

Comment:
What are all the ignored function parameters for?

Reply:
Since `inModelF` has the type `Section -> Section -> Section -> Section -> [Contents] -> Section` as can be witnessed here:
```
inModelF :: Section -> Section -> Section -> Section -> [Contents] -> Section
inModelF probDes datDef theMod genDef otherContents = if not (null otherContents) then 
                                                        SRS.inModel 
                                                          (inModelIntro probDes 
                                                          datDef theMod genDef 
                                                          : otherContents) []
                                                      else
                                                        SRS.inModel 
                                                          (inModelIntroNoContent 
                                                          probDes datDef theMod 
                                                          genDef : 
                                                          otherContents) []
```
 I needed to have four section parameters for `inModelIntroNoContent` so that it can satisfy the type of the function`inModelF`.

Reply:
I think you should be able to omit those parameters seeing as they're not used.

Reply:
What happens if you remove the extra arguments in the function call?